src/content/en/updates/posts/2016/07/infinite-scroller.md

```diff
@@ -0,0 +1,246 @@
+---
+layout: updates/post
+title: "Complexities of an infinite scroller"
+description: "Infinite scrollers are a common UI pattern. We are trying to solve the technical challenges hidden within."
+published_on: 2016-07-01
+updated_on: 2016-07-01
+authors:
+  - surma
+tags:
+  - javascript
+  - css
+featured_image: /web/updates/images/2016/07/infinite-scroller/screenshot.png
+
+---
+
+TL;DR: [Demo][Infscroller Demo] of an infinite scroller, [code][Infscroller Code] for the infinite scroller
```

Comment:
Here's a page that shows the way to do tl:drs https://github.com/google/WebFundamentals/blob/master/src/content/en/fundamentals/design-and-ui/animations/index.markdown